src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -117,11 +117,15 @@ function MetadataSidebarRedesign({
     };
 
     const handleSubmit = async (values: FormValues, operations: JSONPatchOperations) => {
-        isExistingMetadataInstance()
-            ? handleUpdateMetadataInstance(values.metadata as MetadataTemplateInstance, operations, () =>
-                  setEditingTemplate(null),
-              )
-            : handleCreateMetadataInstance(values.metadata as MetadataTemplateInstance, () => setEditingTemplate(null));
+        if (isExistingMetadataInstance()) {
+            await handleUpdateMetadataInstance(values.metadata as MetadataTemplateInstance, operations, () =>
```

Comment:
Can we use [`convertTemplateToTemplateInstance`](https://github.com/wpiesiak/box-ui-elements/blob/metadata-fix-loading/src/elements/content-sidebar/utils/convertTemplateToTemplateInstance.ts) here as it's already merged ?

Reply:
`values.metadata `has all required fields for `MetadataTemplateInstance`, we just need to formally convert the type. There is no reason here to use [convertTemplateToTemplateInstance](https://github.com/wpiesiak/box-ui-elements/blob/metadata-fix-loading/src/elements/content-sidebar/utils/convertTemplateToTemplateInstance.ts) 🤔